tasks/integration/linux.yml

```diff
@@ -9,3 +9,4 @@
 
 - name: Include shared integration installation and removal tasks
   include_tasks: integration/_linux-macos-shared.yml
+  loop: "{{ datadog_integration|dict2items }}"
```

Comment:
What does the output look like when trying to install multiple integrations through `include_tasks` in a loop? Will it log the exact same task name every single time? If so, would it be possible to parametrize the name to make the output more meaningful?